assets/pages/paper-subscription-landing/paperSubscriptionLandingPageActions.js

```diff
@@ -4,11 +4,21 @@
 
 import { type PaperBillingPlan } from 'helpers/subscriptions';
 import { ProductPagePlanFormActionsFor } from 'components/productPage/productPagePlanForm/productPagePlanFormActions';
+import { type PaperDeliveryMethod } from 'helpers/subscriptions';
+import { paperSubsUrl } from 'helpers/routes';
+
+// ----- Types ----- //
+export type TabActions = { type: 'SET_TAB', tab: PaperDeliveryMethod }
+
 
 // ----- Action Creators ----- //
 
 const { setPlan } = ProductPagePlanFormActionsFor<PaperBillingPlan>('Paper', 'Paper');
+const setTab = (tab: PaperDeliveryMethod): TabActions => {
+  window.history.replaceState({}, null, paperSubsUrl(tab));
```

Comment:
What's the thinking behind this?

Reply:
have the url match the selected tab as they both have canonical pages and routes